src/Randomizer.CrossPlatform/Views/MultiRomListPanel.axaml

```diff
@@ -8,7 +8,14 @@
              x:DataType="viewModels:MultiRomListViewModel"
              x:Class="Randomizer.CrossPlatform.Views.MultiRomListPanel">
   <Grid RowDefinitions="*, Auto">
-    <ListBox Grid.Row="0" ItemsSource="{Binding Games}" BorderThickness="0">
+    <StackPanel Grid.Row="0" IsVisible="{Binding !IsListVisible}" Margin="5 2">
+      <TextBlock TextWrapping="Wrap">It looks like you haven't joined or created any multiplayer games! Click Create game to host a name game for others to join, or click Join game to join a multiplayer game created by someone else. Multiplayer games require auto tracking, so verify that is fully setup.</TextBlock>
```

Comment:
This "name game" typo already exists, but we might as well fix it here, I suppose

Reply:
I went ahead and fixed in both places